src/core/streaming.h

```diff
@@ -180,7 +182,53 @@ namespace librealsense
         {
             //empty
         }
-    private:
+
+    protected:
         float m_depth_units;
     };
+
+    class depth_stereo_sensor : public virtual depth_sensor, public recordable<depth_stereo_sensor>
+    {
+    public:
+        virtual float get_stereo_baseline_mm() const = 0;
+        virtual ~depth_stereo_sensor() = default;
```

Comment:
It is not a polymorphic base class => you don't need to specify the default dtor

Reply:
Corrected